github/tests/Project.py

```diff
@@ -0,0 +1,140 @@
+        self.assertEqual(proj.id, id)
+        self.assertEqual(proj.node_id, "MDc6UHJvamVjdDE2ODI5NDE=")
+        self.assertEqual(proj.name, 'TestProject')
+        self.assertEqual(proj.body, "To be used for testing project access API for PyGithub.")
+        self.assertEqual(proj.number, 1)
+        self.assertEqual(proj.state, "open")
+        self.assertEqual(proj.creator, self.repo.owner)
+        self.assertEqual(proj.created_at.year, 2018)
+        self.assertTrue(proj.updated_at > proj.created_at)
+
+    # See https://developer.github.com/v3/projects/columns/#get-a-project-column
+    def testProjectColumnAttributes(self):
+        proj = self.g.get_project(1682941)
+        col = proj.get_columns()[0]
+        self.assertEqual(col.id, 3138830)
+        self.assertEqual(col.node_id, "MDEzOlByb2plY3RDb2x1bW4zMTM4ODMw")
+        self.assertEqual(col.name, "To Do")
+        self.assertEqual(col.url, "https://api.github.com/projects/columns/3138830")
+        self.assertEqual(col.project_url, "https://api.github.com/projects/1682941")
+        self.assertEqual(col.cards_url, "https://api.github.com/projects/columns/3138830/cards")
+        self.assertEqual(col.created_at.year, 2018)
+        self.assertTrue(col.updated_at >= col.created_at)
+
+    # See https://developer.github.com/v3/projects/cards/#get-a-project-card
+    def testProjectCardAttributes(self):
+        proj = self.g.get_project(1682941)
+        col = proj.get_columns()[1]
+        card = col.get_cards()[0]
+        self.assertEqual(card.url, "https://api.github.com/projects/columns/cards/11780055")
+        self.assertEqual(card.column_url, "https://api.github.com/projects/columns/3138831")
+        self.assertEqual(card.content_url, "https://api.github.com/repos/bbi-yggy/PyGithub/issues/1")
+        self.assertEqual(card.id, 11780055)
+        self.assertEqual(card.node_id, "MDExOlByb2plY3RDYXJkMTE3ODAwNTU=")
+        self.assertEqual(card.note, None)   # No notes for cards with content.
+        self.assertEqual(card.creator, self.repo.owner)
+        self.assertEqual(card.created_at.year, 2018)
+        self.assertTrue(card.updated_at >= card.created_at)
+        self.assertFalse(card.archived)
+
+    def testGetProjectCardContent(self):
+        proj = self.g.get_project(1682941)
+        col = proj.get_columns()[1]
+        cards = col.get_cards()
+
+        pull_card = cards[0]
+        pull = pull_card.get_content("PullRequest")
+        self.assertIsInstance(pull, github.PullRequest.PullRequest);
+        self.assertEqual(pull.title, "Work in progress on support for GitHub projects API.")
+
+        issue_card = cards[1]
+        issue = issue_card.get_content()
+        self.assertIsInstance(issue, github.Issue.Issue);
+        self.assertEqual(issue.title, "Test issue")
+
+        note_card = cards[2]
+        note_content = note_card.get_content()
+        self.assertEqual(note_content, None)
+        
+    def testGetAllProjectCards(self):

```

Comment:
Is there a way to generate output from a test when just that test is running, but suppress it when multiple tests are being run? Printing intermediate results would be helpful at time while developing tests.

Reply:
I don't follow. Can't you just run a single test at a time?

Reply:
I meant that while I'm testing my work I would like to know details of what the test is doing - by printing additional information to the output - but when running a test suite I just want to know pass or failure status.